packages/flutter_tools/lib/src/android/device_android.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:async';
 import 'dart:io';
 
 import 'package:crypto/crypto.dart';
+import 'package:flutter_tools/flx.dart' as flx;
```

Comment:
This file is also in the flutter_tools package. Can't you simply refer to it with a relative path?